src/qililab/platform/platform.py

```diff
@@ -1435,111 +1424,73 @@ def execute(
-
-        Finds the absolute order of each measurement for each qubit and its corresponding key in the
-        same format as in qblox's aqcuisitions dictionary (#qubit, #qubit_measurement).
-
-        Then it orders results in the same measurement order as the one in circuit.queue.
-
-        Args:
-            result (Result): Result obtained from the execution
-            circuit (Circuit): Qibo circuit being executed
-            final_layouts (list[int], optional): Final layout of the original logical qubits in the physical circuit:
-                [Logical qubit in wire 1, Logical qubit in wire 2, ...] (None = trivial mapping).
-
-        Returns:
-            Result: Result obtained from the execution, with each measurement in the same order as in circuit.queue.
-        """
-        if not isinstance(result, QbloxResult):
-            raise NotImplementedError("Result ordering is only implemented for qblox results")
-
-        # register the overall order of all qubit measurements.
-        qubits_m = {}
-        order = {}
-        # iterate over qubits measured in same order as they appear in the circuit
-        # TODO: You need to check where each measurement is, since SWAPs can be after a measurement...
-        # FIXME: In the meanwhile do it asuming the Measurement is the last gate for each qubit
-        for i, qubit in enumerate(qubit for gate in circuit.queue for qubit in gate.qubits if isinstance(gate, M)):
-            if qubit not in qubits_m:
-                qubits_m[qubit] = 0
-            order[qubit, qubits_m[qubit]] = i
-            qubits_m[qubit] += 1
-        if len(order) != len(result.qblox_raw_results):
-            raise ValueError(
-                f"Number of measurements in the circuit {len(order)} does not match number of acquisitions {len(result.qblox_raw_results)}"
-            )
+        qprogram = self.compile(circuit, nshots, transpilation_config)
 
-        # Tell users that the final layout is being undone:
-        logger.info(
-            "Undoing final physical qubit mapping, so you get back the original qubit order in your logical circuit."
-        )
+        results = self.execute_qprogram(qprogram)
 
-        # allocate each measurement its corresponding index in the results list
-        results = [None] * len(order)  # type: list | list[dict]
-        for qblox_result in result.qblox_raw_results:
-            measurement = qblox_result["measurement"]
-            physical_qubit = qblox_result["qubit"]
-            original_logical_qubit = final_layout[physical_qubit] if final_layout else physical_qubit
-
-            # TODO:Check this is correct, or how it works with multiple measurements per qubit:
-            original_measure_num = order[original_logical_qubit, measurement]
-            results[original_measure_num] = qblox_result
+        return results
 
-        return QbloxResult(integration_lengths=result.integration_lengths, qblox_raw_results=results)
+    # @staticmethod
```

Comment:
Why are we keeping all this commented code?

Reply:
Removed.